deploy-board/deploy_board/templates/groups/asg_config.html

```diff
@@ -39,6 +40,16 @@
     <div id="asgConfigPid" class="panel panel-default">
         {% include "groups/asg_config.tmpl" %}
     </div>
+
+    <div id="pasConfigPid" class="panel panel-default">
+        {% include "groups/pase_config.tmpl" %}
+    </div>
```

Comment:
I think it's better to put this div in asg_config.tmpl. The alarm or time based auto scaling page is created in asg_config.tmpl based on "autoScalingType". And this pasConfig should be part of alarm based asg.